test/fixtures/hostfuncs/host.ex

```diff
@@ -2,17 +2,17 @@ defmodule Host do
   use WaspVM.HostFunction
 
   defhost function0(_a, _b) do
-    WaspVM.HostFunction.API.update_memory(ctx, "memory1", 0, <<0, 50, 130, 53>>) # 3310133
-    WaspVM.HostFunction.API.update_memory(ctx, "memory1", 32, <<0, 102, 20, 75>>) # 6689867
+    WaspVM.HostFunction.API.update_memory(ctx, "memory1", 0, <<53, 130, 50, 0>>) # 3310133
+    WaspVM.HostFunction.API.update_memory(ctx, "memory1", 32, <<75, 20, 102, 0>>) # 6689867
   end
 
   defhost function1 do
```

Comment:
Is this Legacy/Ready For Removal?

Reply:
I don't believe so, pretty sure these functions are being used by the tests somewhere